src/rtc/h5_prep.py

```diff
@@ -300,20 +309,137 @@ def get_metadata_dict(product_id: str,
+             ['processing_type', processing_type,
              'NOMINAL (or) URGENT (or) CUSTOM (or) UNDEFINED'],
-        # datetime format 'YYYY-MM-DD HH:MM:SS'
         'identification/processingDateTime':
-            ['processing_date_time',
-             datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
-             'Processing date and time'],
+             ['processing_date_time',
+              datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
+              'Processing date and time'],
         'identification/productVersion':
             ['product_version', product_version, 'Product version'],
         'identification/softwareVersion':
-            ['software_version', str(SOFTWARE_VERSION), 'Software version'],
-        # 'identification/CEOSDocumentIdentifier':
-        #     ["https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L-PFS_NRB_v5.5.pdf",
-        #      'CEOS document identifier'],
+             ['software_version', str(SOFTWARE_VERSION), 'Software version'],
+        #'identification/CEOSDocumentIdentifier':
+        #    ["https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L-PFS_NRB_v5.5.pdf",
+        #     'Product version'],
+        'RTC/metadata/sourceDataInformation/numberOfAcquisitions': # 1.6.4
+            ['number_of_acquisitions',
+             1,
+             'Number of source data acquisitions'],
+        # TODO Review: should we expose this parameter in the runconfig?
+        'RTC/metadata/sourceDataInformation/dataAccess':
+            ['data_access',
+             'https://search.asf.alaska.edu/',
+             'Data access URL'],
+        'RTC/metadata/sourceDataInformation/radarBand':  # 1.6.4
+            ['radar_band', 'C', 'Radar band'],
+        'RTC/metadata/sourceDataInformation/processingFacility': #  1.6.6
+            ['source_processing_facility',
+             (f'organization: \"{burst_in.burst_misc_metadata.processing_info_dict["organisation"]}\", '
+              f'site: \"{burst_in.burst_misc_metadata.processing_info_dict["site"]}\", '
+              f'country: \"{burst_in.burst_misc_metadata.processing_info_dict["country"]}\"'),
+             'Source data processing facility'],
+        'RTC/metadata/sourceDataInformation/processingDateTime':  # 1.6.6
+            ['source_processing_date_time',
+             burst_in.burst_misc_metadata.processing_info_dict['stop'],
+             'Processing date and time of the source data'],
+        'RTC/metadata/sourceDataInformation/processingSoftwareVersion':  # 1.6.6
+            ['source_processing_sw_version',
+             str(burst_in.ipf_version),
+             'IPF version of the source data'],
+        
+        'RTC/metadata/sourceDataInformation/azimuthLooks':  # 1.6.6
+            ['source_azimuth_looks',
+             burst_in.burst_misc_metadata.azimuth_looks,
+             'Azimuth number of looks'],
+        'RTC/metadata/sourceDataInformation/slantRangeLooks':  # 1.6.6
+            ['source_slant_range_looks',
```

Comment:
```suggestion
            [None,
```
Let's also remove this from the GeoTIFF metadata